### PR TITLE
[PC-1088] Portenta C33 (Post-launch review)

### PR DESCRIPTION
**Pull request by Jorgetrujilloroman**
## What This PR Changes
-Small typos and grammar fixes on the product page.

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Review by jcarolinares — CHANGES_REQUESTED** [review]
Please check the order of table footnotes too

image

File: content/hardware/04.pro/boards/portenta-c33/essentials.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
         The Wi-Fi® library is designed to use the ESP32-C3-MINI-1U module, which allows your Arduino to connect to the Internet.    </EssentialElement>
 
 <EssentialElement link="https://www.arduino.cc/en/Reference/ArduinoBLE" title="Arduino BLE" type="library">
-    Bluetooth® 4.0 includes both traditional Bluetooth®, now labeled " Bluetooth® Classic", and the Bluetooth® Low Energy.
+    Bluetooth® 5.0 includes both traditional Bluetooth®, now labeled " Bluetooth® Classic", and Bluetooth® Low Energy.
```
Comment (jcarolinares) [inline content/hardware/04.pro/boards/portenta-c33/essentials.md:13]: Good cath but I'm afraid you are wrong. Let me explain:

* Most of our recent boards are Bluetooth 5.0 by hardware
* But the Library Arduino BLE it still not compatible with Bluetooth 5.0.
Please only apply the removal of the "the"

**Review by jcarolinares — CHANGES_REQUESTED** [review]
Please proceed with the small change/correction

File: content/hardware/04.pro/boards/portenta-c33/essentials.md (hunk 1 above)
Comment (Jorgetrujilloroman) [inline content/hardware/04.pro/boards/portenta-c33/essentials.md:13]:
```suggestion
    Bluetooth® 4.0 includes both traditional Bluetooth®, now labeled " Bluetooth® Classic", and Bluetooth® Low Energy.

```

Comment (Jorgetrujilloroman) [reply content/hardware/04.pro/boards/portenta-c33/essentials.md:13]: @jcarolinares

**Review by jcarolinares — APPROVED** [review]
Looks good to me

Once the CI has finished do the merge and check when it published that the change are on the server with no problems please.

thank you!

**Pull request merged**